docker/Dockerfile.olbase

```diff
@@ -45,6 +48,10 @@ WORKDIR /openlibrary
 
 RUN pip install -r requirements_common.txt
 
+RUN mkdir /node_modules
+RUN mv node_modules node_modules.bak
+RUN ln -s /node_modules
```

Comment:
On testing in docker, I encountered an error in building `olbase` on these lines and it failed on "cannot stat node_modules".

I removed 51-53 and it worked. I'm not quite sure what you were doing with creating the dir, I imagine there is something we could do to allow node dependencies to update cleanly/efficiently in the docker layers?

Reply:
Thanks for the review! Happy to explain...

The working directory is the mounted host file system, so node_modules contains any modules installed on the host or other virtual images (e.g. Vagrant). I could just blow it away, but it seemed more conservative to rename it.

`ln -s` without an origin defaults to the current directory. The goal is to have host/openlibrary/node_modules be a soft link to a directory in the currently running container (which, obviously, is in a consistent location across all containers, so I'd thought of making it something like ../ol_modules instead of /node_modules

This extra machinery really only exists to deal with corner case, e.g. I run an OS X host and Linux VM, so anyone running a homogenous environment w/o Window, OS X, multiple CPU ISAs/architectures is probably fine.